packages/office-ui-fabric-react/src/components/DetailsList/DetailsList.base.tsx

```diff
@@ -769,7 +772,7 @@ export class DetailsListBase extends BaseComponent<IDetailsListProps, IDetailsLi
     firstIndex: number
   ): IColumn[] {
     const { selectionMode, checkboxVisibility, groups } = props;
-    const outerPadding = DEFAULT_INNER_PADDING;
+    const outerPadding = DEFAULT_ROW_HORIZONTAL_PADDING * 2;
```

Comment:
Not too familiar with this bit here, but since we now pass these in as props, should we then use props.rowHorizontalPadding here?

Reply:
@kenotron In `DetailsList.base.tsx` they are coming from simply importing them from the `DetailsRow.styles` and not passed by props)))